CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ incremental in minor, bugfixes only are patches.
 See [0Ver](https://0ver.org/).
 
 
+## 0.23.0
+
+### Misc
+
+- *Breaking*: Remove `Result.success_type` and `Result.failure_type`
```

Comment:
Let's list all removed items in a single point

Reply:
I made the change in a new commit